src/est/import/lib/EFI/Import/Metadata.pm

```diff
@@ -24,11 +25,13 @@ use constant FIELD_UNIREF50_CLUSTER_SIZE => "UniRef50_Cluster_Size";
 use constant FIELD_UNIREF90_CLUSTER_SIZE => "UniRef90_Cluster_Size";
 use constant FIELD_UNIREF100_CLUSTER_SIZE => "UniRef100_Cluster_Size";
 
+use constant INPUT_SEQ_ID => "zINPUTSEQ";
```

Comment:
I think BLAST uppercases all accession IDs, lets make this uppercase as well so that we don't have the possibility of losing a reference somewhere. This caused the pipeline to fail because some alignment scores were null.

Reply:
How does it work in the FASTA option, where the sequences are prefixed with lowercase 'zzz's?

Reply:
I'm not sure. I saw this discrepancy between the transcoded FASTA (zINPUTSEQ) file and the output of the axa_blast stage (ZINPUTSEQ). I'll trace it through a little more and see whats going on

Reply:
The testing we have done on FASTA files has only included sequences which are in our test database. I think this means that the accession IDs are not rewritten and so that is why we haven't seen this problem yet.

UniProt [defines their accession ID format](https://www.uniprot.org/help/accession_numbers) to include only uppercase letters. I think we should follow that convention when we need to rewrite sequences, because it seems like BLAST does. I will add a test that uses a FASTA file with unknown Accession IDs to catch this in the future.

Reply:
Unknown sequences were identified by a lowercase 'z' at the start of the ID.  If it's uppercase then the unknown sequence ID checks need to be updated everywhere.  UniProt includes sequences that start with a Z, but we can still identify unknown sequences by checking for consecutive ZZs, since UniProt is always A-Z at the start followed by a number.

Reply:
Can we avoid storing information about UniProt membership in the sequence altogether?